pkg/helpers/upgrade.go

```diff
@@ -0,0 +1,88 @@
+package helpers
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/pior/dad/pkg/config"
+	"github.com/pior/dad/pkg/executor"
+)
+
+type Upgrade struct {
+	github    *Github
+	plateform string
+	client    *http.Client
+	skipSudo  bool
```

Comment:
that's a nasty trick being used for testing here https://github.com/pior/dad/pull/50/files#diff-8c7178b8918e3df7e27e5a83c66b4c2cR71

Not sure how what would be the best approach to skip sudo when testing...

Reply:
I'm not sure.
I think we could reverse the meaning of it for more "regular" `useSudo`.
(not that make it really better)